WordPress/src/main/res/layout/notifications_list_fragment.xml

```diff
@@ -50,6 +50,7 @@
         android:layout_height="match_parent"
         android:layout_marginTop="@dimen/toolbar_height"
         android:gravity="center"
+        android:background="@drawable/back"
```

Comment:
Suggestion: We can rename `back` with a generic name like `bg_rectangle_white_stroke_black` to be consistent with other color drawables. But wait to rename it till the color is decided.